third_party/move/move-compiler-v2/tests/checking/variants/variants_test_err.move

```diff
@@ -0,0 +1,35 @@
+module 0x815::m {
+
+    enum Color {
+        RGB{red: u64, green: u64, blue: u64},
+        Red,
+        Blue,
+    }
+
+    enum Shape {
+        Quadrant{length: u64},
+        Circle{radius: u64}
+
+    }
+
+    fun test1(c: Color): bool {
+        (c is Red|Circle)
+    }
+
+    fun test2(c: Color): bool {
+        (c is Red|Shape::Circle)
```

Comment:
Can we add a test of a qualified one that works?  (E.g., `Color::Red`)?

Reply:
Those tests are in variant _tests_ok.move, also with generics.